options/m_option.h

```diff
@@ -256,6 +256,8 @@ union m_option_value {
     struct m_channels channels;
 };
 
+static const union m_option_value m_option_value_default;
```

Comment:
Small nit: I'd add a small comment on the union `{0}` situation here, since this is a bit unusual at a glance.

Otherwise LGTM.

Reply:
I was thinking something shorter like "fully zeroed due to static storage. unlike `{0}` which only zeros the first union member".

But this also works, so 👍 by me.